Stripe/STPCustomer.m

```diff
@@ -8,15 +8,23 @@
 
 #import "STPCustomer.h"
 
+#import "STPAddress.h"
 #import "STPCard.h"
 #import "STPSource.h"
 #import "StripeError.h"
+#import "NSDictionary+Stripe.h"
 
+/**
+ NOTE: STPCustomer has been deprecated. When we remove STPBackendAPIAdapter,
+ we should rename STPCustomer with STPLimitedCustomer, to avoid confusion –
+ customers retrieved using an ephemeral key have a limited set of fields.
```

Comment:
Not sure this rename is really needed, especially given that most people probably aren't using it except through payment context?